examples/run_emodel/run_emodel.py

```diff
@@ -0,0 +1,186 @@
+"""Running the emodel with BlueCelluLab"""
+
+import matplotlib.pyplot as plt
```

Comment:
Could you re-arrange the order of the imports? You can use `isort --check-only --sl --diff run_emodel.py` to have something cleaner